pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -363,6 +372,151 @@ func testTLSClassification(t *testing.T, tr *Tracer, clientHost, targetHost, ser
 	})
 }
 
+func testTLSAMQPClassification(t *testing.T, tr *Tracer, _, targetHost, serverHost string) {
+	skipFunc := composeSkips(skipIfNotLinux, skipIfUsingNAT, skipIfHTTPSNotSupported)
```

Comment:
nit: Change `skipIfHTTPSNotSupported` to `skipIfTLSNotSupported`

that can happen in a followup PR

Reply:
I am in favor of doing this change in a separate PR, since we have a few places in our code that uses "HTTPS support" instead of "TLS support", not only in the tests.